source/include/dqm4hep/StatusCodes.h

```diff
@@ -36,6 +36,19 @@
 #include <execinfo.h>
 #endif
 
+#define SOFT_RETURN_RESULT_IF(Command)                                                                                \
```

Comment:
For the use you make, you don't need this macro. Use the `RETURN_RESULT_IF_AND_IF` macro as:
```cpp
RETURN_RESULT_IF_AND_IF(STATUS_CODE_SUCCESS, STATUS_CODE_NOT_FOUND, !=, operation);
```